auto generated update of versioned plugin documentation (with fix) (#1743)
elastic/logstash-docs: docs/versioned-plugins/inputs/google_pubsub-v1.4.0.asciidoc

https://cloud.google.com/ to get started:

- Google Cloud Platform Projects and https://cloud.google.com/docs/overview/[Overview]
- Google Cloud Pub/Sub https://cloud.google.com/pubsub/[documentation]
- Stackdriver Logging https://cloud.google.com/logging/[documentation]

==== Cloud Pub/Sub

Currently, this module requires you to create a `topic` manually and specify
it in the logstash config file. You must also specify a `subscription`, but
the plugin will attempt to create the pull-based `subscription` on its own.

All messages received from Pub/Sub will be converted to a logstash `event`
and added to the processing pipeline queue. All Pub/Sub messages will be
`acknowledged` and removed from the Pub/Sub `topic` (please see more about
https://cloud.google.com/pubsub/overview#concepts)[Pub/Sub concepts].

It is generally assumed that incoming messages will be in JSON and added to
the logstash `event` as-is. However, if a plain text message is received, the
plugin will return the raw text in as `raw_message` in the logstash `event`.

==== Authentication

You have two options for authentication depending on where you run Logstash.

> 1. If you are running Logstash outside of Google Cloud Platform, then you will
> path to the JSON private key file in your config in `json_key_file` setting.

Reviewer: ```suggestion
1. If you are running Logstash outside of Google Cloud Platform, you need to
provide the path to the JSON private key file in your config in `json_key_file` setting.
```
